use-timescale/compression/compression-policy.md

```diff
@@ -61,7 +61,11 @@ For more information, see the API reference for [`timescaledb_information.jobs`]
 
 ## Pause compression policy
 
-To disable a compression policy temporarily:
+To disable a compression policy temporarily, find the corresponding job ID and then call `alter_job` to pause it:
+
+```sql
+SELECT * FROM timescaledb_information.jobs where proc_name = 'policy_compression' AND relname = ''
+```
```

Comment:
Sorry I wasn't clear, relname would be set to the hypertable name. So however we show that in other places in the docs `<hypertablename>` maybe?

Reply:
Oooh, missed that, sorry. This page seems to be using both - placeholders and examples - so I aligned all the commands to refer to the same hypertable called "example". Hope this works.